chart/compass/values.yaml

```diff
@@ -637,6 +642,7 @@ global:
     systemsAPIFilterCriteria: ""
     # systemToTemplateMappings - how to map system properties to an existing application template
     systemToTemplateMappings: '{}'
+    templateRegion: "eu-1"
```

Comment:
Let's discuss this workaround again

Reply:
after an offline discussion we've decided to implement app template creation on system-fetcher start as next step